internal/resourcemodifiers/resource_modifiers.go

```diff
@@ -117,7 +118,24 @@ func (r *ResourceModifierRule) PatchArrayToByteArray() ([]byte, error) {
 }
 
 func (p *JSONPatch) ToString() string {
-	if strings.Contains(p.Value, "\"") {
+	// if value is empty, then add empty quotes
+	if p.Value == "" {
```

Comment:
I would remove this first if, if falls back to implementation of line 141, right?

Reply:
I would personally prefer this edge case handling
in case the integer check or some other check has handling for "" and maps it to default value.
this just gives sanity.